wgsl/index.bs

```diff
@@ -7724,9 +7724,10 @@ the next statement until the end of the `body`.
 The declaration is executed each time it is reached, so each new iteration
 creates a new instance of the variable or constant, and re-initializes it.
 
-<div class='example glsl' heading="For to Loop transformation">
+<div class='example wgsl function-scope' heading="For to Loop transformation">
   <xmp>
-    for(var i: i32 = 0; i < 4; i++) {
+    var a: i32 = 2;
+    for (var i: i32 = 0; i < 4; i++) {
```

Comment:
All other uses of `for` in the spec has a space between `for` and `(` so I did the same here.